src/index.js

```diff
@@ -1,78 +1,226 @@
+import AppBar from './app-bar';
```

Comment:
Hello, I see that 1 line export is not considered a good practice. https://github.com/airbnb/javascript#10.3